modules/runtime/src/service/systemd/xp.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=Enonic XP
+Documentation=https://developer.enonic.com/docs
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=simple
+PrivateTmp=true
+Environment=XP_INSTALL=/opt/enonic/xp
+Environment=XP_JAVA_HOME=${XP_INSTALL}/jdk
+
+WorkingDirectory=/opt/enonic/xp
```

Comment:
`/opt` Is it commonly used directory nowadays?

Reply:
```
/opt – Optional Packages

The /opt directory contains subdirectories for optional software packages. It’s commonly used by proprietary software that doesn’t obey the standard file system hierarchy – for example, a proprietary program might dump its files in /opt/application when you install it.
```

So if the XP distro is installed there this makes sense.